tests/support/peerManager.ts

```diff
@@ -312,10 +333,23 @@ export class RadiclePeer {
   }
 
   public uiUrl(): string {
-    return `/seeds/127.0.0.1:8080`;
+    if (!this.#httpdBaseUrl) {
+      throw new Error("No httpd service running");
+    }
+
+    return `/seeds/${this.#httpdBaseUrl.hostname}`;
```

Comment:
Shouldn’t this be 
```suggestion
    return `/seeds/${this.#httpdBaseUrl.hostname}:${this.#httpdBaseUrl.port}`;
```

Reply:
Yeah probably better since we eventually use random ports

Reply:
Done ✅